web/client/actions/__tests__/mousePosition-test.js

```diff
@@ -11,9 +11,13 @@ var {
     CHANGE_MOUSE_POSITION,
     CHANGE_MOUSE_POSITION_CRS,
     CHANGE_MOUSE_POSITION_STATE,
+    MOUSE_MOVE_MAP_EVENT,
+    MOUSE_OUT,
     changeMousePosition,
     changeMousePositionCrs,
-    changeMousePositionState
+    changeMousePositionState,
+    mouseMoveMapEvent,
+    mouseOut
```

Comment:
Maybe I wasn't clear: 
- mouseposition (actions/reducers etc..) are all related to mouseposition ONLY plugin
- mouseout, mousemove are actions (events) from map plugin
You can intercept mouse events (Actions) from mouseposition plugin (epic, reducer) or identify plugin (action /reducers)  but the actions are proper of the map plugin (so they should go in map actions file

Reply:
Done